net-irc/soju/files/soju.tmpfiles

```diff
@@ -0,0 +1 @@
+d /run/soju 0750 soju soju -
```

Comment:
Fix EOF

Reply:
Hm, thought Emacs did this automatically. Fixed

Reply:
Also I removed the '-' since I guess I want auto cleanup here.